src/clients/apiClient.ts

```diff
@@ -1,12 +1,36 @@
 export const backendUrl = import.meta.env.VITE_BACKEND_URL;
 
+export interface NewUser {
+  username: string;
+  password: string;
+  userBio: string;
+  profileImageUrl: string;
+}
+
 export const checkBackendConnection = async (): Promise<boolean> => {
   let response: Response;
   try {
     response = await fetch(`${backendUrl}/test`);
   } catch {
     return false;
   }
-  
   return response.ok;
 }
+
+export async function createNewUser(newUser: NewUser): Promise<Response> {
+  let response: Response;
+
+  response = await fetch(`${backendUrl}/users/create`, {
```

Comment:
we can make this a single line
```suggestion
  const response = await fetch(`${backendUrl}/users/create`, {
```

Reply:
changes made